apps/ejabberd/src/foreign_event/mod_foreign_http.erl

```diff
@@ -28,9 +28,11 @@
 %%--------------------------------------------------------------------
 
 make_request(Request, OnResponse) ->
-    {Host, Path, Method, Headers, Body} = parse(Request),
-    Response = do_make_request(Host, Path, Method, Headers, Body),
-    respond(encode(Response), OnResponse),
+    spawn(fun() ->
```

Comment:
There will be a pool for that :) This PR introduced only IQ handler, `spawn` is just a placeholder. There will be aconfigurable pool later :)